src/main/java/org/candlepin/pki/certs/AnonymousCertificateGenerator.java

```diff
@@ -317,16 +313,15 @@ private Set<X509Extension> prepareV3ByteExtensions(org.candlepin.model.dto.Produ
     }
 
     private byte[] createContentAccessDataPayload(Consumer consumer,
-        Map<org.candlepin.model.Content, Boolean> ownerContent, PromotedContent promotedContent) {
+        Map<String, ProductContent> activeContent, PromotedContent promotedContent) {
 
         String consumerUuid = consumer != null ? consumer.getUuid() : null;
         log.info("Generating SCA payload for consumer \"{}\"...", consumerUuid);
 
         Product engProduct = new Product()
             .setId("content_access")
-            .setName(" Content Access");
-
-        ownerContent.forEach(engProduct::addContent);
+            .setName(" Content Access")
+            .setProductContent(activeContent.values());
```

Comment:
Can we update `content_access` and ` Content Access` here into something more accurate, like `anonymous_cloud_offering_access`/`Anonymous Cloud Offering Access`?

And same thing for the `skuProduct` object below this?

Reply:
Also, let me know if you can think of a more accurate name. @bakajstep @Ceiu @joshmalbrecht Once the cert implementation is out, its hard to change because people start parsing it. 

I thought only adding 'Anonymous Access' isn't specific enough, because there's been talk of eventually introucing anonymous/automatic registration for non-cloud systems. Perhaps `Anonymous Cloud Access` is enough (without the Offering part)?

Reply:
I like `Anonymous Cloud Content Access` the most

Reply:
We talked about it at scrum and agreed on `Anonymous Cloud Content Access`

Reply:
done